docs/pages/styles/type.html

```diff
@@ -11,13 +11,13 @@
 
     <h2>Sizes</h2>
 
-    <p><strong>5</strong> —&nbsp;Dosis Semi 45px | Line-height 52px</p>
+    <p><strong>5</strong> —&nbsp;Roboto Semi 45px | Line-height 52px</p>
```

Comment:
There is no "Semi" weight for Roboto, this should be "Medium" — https://fonts.google.com/specimen/Roboto?selection.family=Roboto

There is also a page at http://localhost:3030/pages/styles that should be updated.